sentinel-adapter/sentinel-motan-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/motan/MotanUtils.java

```diff
@@ -50,7 +51,7 @@ public static String getMethodResourceName(Caller<?> caller, Request request, Bo
                 isFirst = false;
             }
         } catch (ClassNotFoundException e) {
-            e.printStackTrace();
+            RecordLog.info("[MotanUtils] getMethodResourceName encountered Exception", e);
```

Comment:
A typo? usually using warn or error level to log exception.

Reply:
yes, it should be in warn level.